vision_evaluation/evaluators.py

```diff
@@ -435,7 +509,10 @@ def reset(self):
 
 class MeanAveragePrecisionEvaluatorForMultipleIOUs(EvaluatorAggregator):
     DEFAULT_IOU_VALUES = [0.3, 0.5, 0.75, 0.9]
+    REPORT_PER_CLASS = [False, False, False, False]
 
-    def __init__(self, ious=DEFAULT_IOU_VALUES):
-        evaluators = [MeanAveragePrecisionEvaluatorForSingleIOU(iou) for iou in ious]
+    def __init__(self, ious=DEFAULT_IOU_VALUES, report_per_class=REPORT_PER_CLASS):
```

Comment:
I don't see a usage where we need to specify different "report_per_class" for different IOUs. How about using a single boolean value to specify it for all IOUs?

Reply:
sync offline, they need to have different boolean value to control